services/src/main/java/org/keycloak/authentication/AuthenticationProcessor.java

```diff
@@ -855,25 +872,33 @@ public Response handleBrowserException(Exception failure) {
             } else if (e.getError() == AuthenticationFlowError.DISPLAY_NOT_SUPPORTED) {
                 ServicesLogger.LOGGER.failedAuthentication(e);
                 event.error(Errors.DISPLAY_UNSUPPORTED);
-                if (e.getResponse() != null) return e.getResponse();
+                if (e.getResponse() != null) {
+                    return e.getResponse();
+                }
                 return ErrorPage.error(session, authenticationSession, Response.Status.BAD_REQUEST, Messages.DISPLAY_UNSUPPORTED);
             } else if (e.getError() == AuthenticationFlowError.CREDENTIAL_SETUP_REQUIRED){
                 ServicesLogger.LOGGER.failedAuthentication(e);
                 event.error(Errors.INVALID_USER_CREDENTIALS);
-                if (e.getResponse() != null) return e.getResponse();
+                if (e.getResponse() != null) {
+                    return e.getResponse();
+                }
                 return ErrorPage.error(session, authenticationSession, Response.Status.BAD_REQUEST, Messages.CREDENTIAL_SETUP_REQUIRED);
             } else if (e.getError() == AuthenticationFlowError.GENERIC_AUTHENTICATION_ERROR) {
                 ServicesLogger.LOGGER.failedAuthentication(e);
                 if (e.getEventDetails() != null) {
                     event.detail(Details.AUTHENTICATION_ERROR_DETAIL, e.getEventDetails());
                 }
                 event.error(Errors.GENERIC_AUTHENTICATION_ERROR);
-                if (e.getResponse() != null) return e.getResponse();
+                if (e.getResponse() != null) {
+                    return e.getResponse();
+                }
                 return ErrorPage.error(session, authenticationSession, Response.Status.BAD_REQUEST, e.getUserErrorMessage());
             } else {
                 ServicesLogger.LOGGER.failedAuthentication(e);
                 event.error(Errors.INVALID_USER_CREDENTIALS);
-                if (e.getResponse() != null) return e.getResponse();
+                if (e.getResponse() != null) {
+                    return e.getResponse();
+                }
```

Comment:
I'm sorry, I haven't yet fully reviewed the PR but I wonder if we could limit unrelated changes like this (splitting conditions to multiple lines) in the PR. It adds some additional noise and makes it a bit harder to review. We could do some refactor like  that as a follow-up. Thanks.

Reply:
Sure, it's just happening because of an automatic setting in the IDE. Do you want me to remove them?

Reply:
Yes please, if you don't mind.

Reply:
Removed and the additional javadocs added.